README.md

```diff
@@ -1,18 +1,58 @@
 # On Prem
 
-## Installing microk8s
+## Installing microk8s and tensorleap from the repo
 
-```
+### On Mac
+
+```bash
+# install microk8s
 brew install ubuntu/microk8s/microk8s
 microk8s install
+microk8s status --wait-ready
 microk8s enable ingress dns storage rbac
+
+# Update /etc/hosts
+echo "$(multipass list --format json | jq -r '.list[] | select(.name == "microk8s-vm") | .ipv4 | first')\ttensorleap.local" | sudo tee -a /etc/hosts
+
+# Add microk8s context to your local KubeConfig file
+KUBECONFIG=<(microk8s config):$HOME/.kube/config kubectl config view --raw > kubeConfig && mv kubeConfig $HOME/.kube/config
+
+# install kapp
+brew tap vmware-tanzu/carvel
+brew install kapp
+
+# install helm and add repositories
+brew install helm
+helm repo add elastic https://helm.elastic.co
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add minio https://charts.min.io/
+
+# Build manifests
+npm install
+npm run build
+
+## ========= MAKE SURE YOU'RE IN MICROK8S CONTEXT ===========
+kubectx -c # => should print microk8s
+
+# get ops service account secret and deploy it to microk8s cluster
+gsutil cp gs://tensorleap-infra-nonprod/ops/default.tfstate - | jq -r '.resources[] | select(.name == "ops_key" and .type == "github_actions_organization_secret") | .instances[].attributes.plaintext_value' > ./json-key-file.json
+kubectl --kubeconfig=<(microk8s config) create secret docker-registry gcr-access-token \
+  --docker-server=gcr.io \
+  --docker-username=_json_key \
+  --docker-password="$(cat ./json-key-file.json)" \
+  --docker-email=someone@tensorleap.ai
+rm ./json-key-file.json
+
+# deploy tensorleap to local cluster
+kubectl apply -f ./dist
 ```
 
+Tensorleap is served in http://tensorleap.local \
 Create a user using signup requests in http://tensorleap.local/api/v2/swagger
 
 ### On Ubuntu
 
-```
+```bash
 sudo apt update
 sudo apt upgrade
```

Comment:
I think that it would be best to use only `apt` or only `apt-get` - not both

Reply:
I agree, but this is what we did and worked
Next time I'm installing a laptop from scratch I'll try to change after it is tested